pkg/util/limiter/rate_limiter_test.go

```diff
@@ -127,3 +163,7 @@ func (s *staticLimitStrategy) Burst(tenantID string) int {
 
 	return tenant.burst
 }
+
+func OK(ok bool, r Reservation) bool {
```

Comment:
No need to expose it. You can rename it to `isOK()` so that doesn't start with an uppercase letter and doesn't clash with `ok` variable too.